Differentiate between authoritative and courtesy translations (#999)
publiccodenet/standard: criteria/use-plain-english.md

* All codebase documentation MUST be in English.
* All [source code](../glossary.md#source-code) MUST be in English, except where [policy](../glossary.md#policy) is machine interpreted as code.
* All bundled policy not available in English MUST have an accompanying summary in English.

> * If the codebase maintainers have committed to providing any translations, then those translation MUST be up to date with the English version and vice versa.

Reviewer: ```suggestion
* If the codebase maintainers have committed to providing any authoritative translations, then those translation MUST be up to date with the English version and vice versa.
```

> * If the codebase maintainers have committed to providing any translations, then those translation MUST be up to date with the English version and vice versa.

Reviewer 2: Let's try to avoid considering English as the unique reference 🙂 There is a list of authoritative, committed languages, that must contain English, but English does not have to be the only reference.

```suggestion
* If the codebase maintainers have committed to providing authoritative any translations, then those translations MUST contain the same content across languages.
```

* Confirm that codebase documentation is available in English.
* Confirm that source code is in English, or confirm any non-English is policy or terms with preceding explanations.

> * Confirm any non-English policy has an accompanying summary in English.

Reviewer 2: ```suggestion
* Confirm any policy has content, or an accompanying summary, in all the authoritative languages.
```

> * Confirm that translations committed to by the maintainers and the English version have the same content.

Reviewer 2: ```suggestion
* Confirm that all authoritative translations have the same content.
```

* Frequently test with policy makers and managers if they understand what you are delivering and how you document it.
* Ask someone outside of your context if they understand the content (for example, a developer working on a different codebase).

> * If there are both required translations and "best effort" community contributed translations, then ensure that it is clearly documented which category each translation belongs to.

Reviewer: ```suggestion
* If there are both authoritative translations and courtesy or automatic (AI) translations, then ensure that it is clearly documented which category each translation belongs to.
```

> * If there are both required translations and "best effort" community contributed translations, then ensure that it is clearly documented which category each translation belongs to.

Reviewer 2: ```suggestion
* If there are both authoritative translations and courtesy translations, then clearly label each translation accordingly.
```
